src/authentication/protected-routes.handler.ts

```diff
@@ -43,7 +32,7 @@ export const isAdminRoute = (url: string, adminRootPath: string): boolean => {
     .map((route) => convertToExpressRoute(route.path))
     .filter((route) => route !== "");
 
-  let urlWithoutAdminRootPath = url;
+  let urlWithoutAdminRootPath = url.split("?")[0];
```

Comment:
[OPTIONAL]

Looks weird. Could work sth more less as:
```
 const { host } = new URL(url);
```

Reply:
`new URL(url);` won't work because `req.originalUrl` and `req.url` contain only `pathname`. I'd have to build a complete url using `req.protocol` and `req.hostname` but `req.hostname` doesn't seem to contain `:port` in localhost.